### DISCUSSION TITLE
Help calibration touch 3,5inch SPI TFT Display ILI9488 + ESP32

### DISCUSSION THREAD

**Discussion by vakulalenko** [Bodmer/TFT_eSPI#2324] [General]
Hello Bodmer,
I'm trying to connect a 3.5-inch SPI TFT touch screen to an Esp 32.
The image is displayed normally, and the sensor seems to be working, but, oddly enough, the point of contact is constantly moving away by a few millimeters and, moreover, in different directions, depending on which way to use the stylus, although in the center it is almost perfect. There is a kind of dead zone on one side of the sensor edge, which should be decently pressed when touching the work.It is connected according to this scheme, with the exception of contacts - T_IRQ also, if necessary, the link where I bought it.I have changed almost nothing in the configuration files, except for the driver type.

https://aliexpress.ru/item/32985467436.html?sub=1010658&af=739_1010658&utm_campaign=1010658&aff_platform=api-new-link-generate&utm_medium=cpa&sub1=1068989&sku_id=66802769933&cn=20ruroi4c27xqdgn50dlr1w9e2gzmav9&dp=20ruroi4c27xqdgn50dlr1w9e2gzmav9&aff_fcid=594a1487f430483083382b793e7a8520-1673742962941-04713-_DnN8zM7&cv=3&aff_fsk=_DnN8zM7&afSmartRedirect=y&sk=_DnN8zM7&aff_trace_key=594a1487f430483083382b793e7a8520-1673742962941-04713-_DnN8zM7&terminal_id=542388cdb8a244948031501f50be8bf3&utm_source=admitad&utm_content=1068989

![image](https://user-images.githubusercontent.com/110564666/212503192-a33bafe3-00c6-46d2-b8fd-49dd41ba1419.png)

**Comment by Bodmer** [comment]
I have one display that has the same symtoms. It seems that occasionally (maybe a manufacturing problem) the screens are quite insensitive to touch and need a much higher pressure to activate. You may find this [alternative library](https://github.com/achillhasler/TFT_eTouch) will work better in these cases as it uses more sophisticated filter algorithms.
